analyses/molecular-subtyping-LGAT/02-subset-fusion-files-LGAT.Rmd

```diff
@@ -0,0 +1,152 @@
+---
+title: "Annotate Fusion defining subtype status for LGAT biospecimens"
+output: html_notebook
+author: K S Gaonkar 
+date: 2020
+---
+
+As per [issue](https://github.com/AlexsLemonade/OpenPBTA-analysis/issues/790) we will be subtyping LGAT based on fusion  in the following genes:
+
+ - LGG, KIAA1549-BRAF
+contains KIAA1549-BRAF fusion
+
+ - LGG, other MAPK
+contains non-canonical BRAF fusion other than KIAA1549-BRAF
+contains RAF1 fusion
+
+- LGG, RTK
+harbors a fusion in ALK, ROS1, NTRK1, NTRK2, or NTRK3 or
+harbors a PDGFRA fusion
+
+- LGG, FGFR
+harbors FGFR1-TACC1 fusion 
+harbors FGFR1 or FGFR2 fusions
+
+- LGG, MYB/MYBL1
+harbors either a MYB-QKI fusion or other MYB or MYBL1 fusion
+
+
+### Setup
+```{r}
+library(tidyverse)
+
+# Look for git root folder
+root_dir <- rprojroot::find_root(rprojroot::has_dir(".git"))
+
+# get subset folder
+subset_dir <- file.path(root_dir, 
+                        "analyses",
+                        "molecular-subtyping-LGAT",
+                        "lgat-subset")
+
+# create if doesn't exist
+if (!dir.exists(subset_dir)) {
+  dir.create(subset_dir)
+}
+```
+
+### Input
+
+`lgat_metadata.tsv` from 00-LGAT-select-pathology-dx that is used for the pathology diagnosis was used to subset clinical file.
+
+`fusion_summary_lgat_foi.tsv` was generated by `fusion-summary` module for LGAT RNA biospecimens.
+
+All fusions in the above gene of interest per subtype was summarized "1" for present and "0" if absent.
+
+For kinase fusions, the following conditions needed to be satisfied for LGAT as per [PR](https://github.com/AlexsLemonade/OpenPBTA-analysis/pull/830):
+- Added all 3' kinase fusions which are in-frame and retain the kinase domain
+- For 5' kinase fusions, added those which are in-frame and retain the kinase domain
+- For 5' kinase fusions that don't meet 3., check whether it has a reciprocal fusion and whether that reciprocal is in-frame and the kinase domain is retained. Then, add those 5' kinase fusions.
+
```

Comment:
Checking my understanding because I did not review #830 - the checks on kinase domain described here currently take place in `fusion-summary` is that correct? If so, can we add that to the text here please?

Reply:
Right, I reorganized the text to make that clear that these rules were applied in fusion-summary and output file is being used in this notebook